packages/graph/README.md

```diff
@@ -13,6 +13,7 @@ The following sections document the hosted services where the subgraph is deploy
 
 | Network        | GITHUB_USER/SUBGRAPH_NAME                    | Playground                                                                                | Query                                                                                 |
 |----------------|----------------------------------------------|-------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| mainnet        | grants-round-mainnet                         | https://thegraph.com/studio/subgraph/grants-round-mainnet               | https://api.studio.thegraph.com/query/14975/grants-round-mainnet/v0.1                 |
```

Comment:
I do not think this the correct query URL.

Reply:
Should query url be something like: https://api.thegraph.com/subgraphs/name/gitcoinco/grants-round-mainnet/ ...?